File: ulti-server/complete/src/main/java/hu/ulti/server/model/Call.java
package hu.ulti.server.model;

public class Call {

	private int callId;
	private Kontra kontra;

	public Call(int callId, Kontra kontra) {
		this.callId = callId;
		this.kontra = kontra;
	}

	public int getCallId() {
		return callId;
	}

	public void setCallId(int callId) {
		this.callId = callId;
	}

	public Kontra getKontra() {
		return kontra;
	}

	public void setKontra(Kontra kontra) {
		this.kontra = kontra;
	}
}

File: ulti-client-react/src/model/call.ts
import { KeyboardEventHandler } from "react";
import { Kontra } from "./kontra";

export class Call {
    public callId: number;
    public kontra: Array<Kontra>;

    public constructor(callId: number, kontra: Array<Kontra>) {
        this.callId = callId;
        this.kontra = kontra;
    }
}